helpme/templates/helpme/ticket_detail.html

```diff
@@ -60,7 +60,7 @@ <h3 class="mt-3">
       {% if comment.user == user %}
       <div class="row mb-3">
 	<div class="col offset-1">
-	  <div class="alert my-message mb-0">
+	  <div class="alert text-light mb-0" style="background: #38629D; border-radius: 15px 15px 0px 15px;">
```

Comment:
This is probably a good place to use the rounded borders.
I'd suggest for the colors use ".bg-info" and ". bg-light" instead of explicitly setting colors.